### PR TITLE
Rename "Sound" to "Sfx"

### PR DESCRIPTION
**Pull request by engineer124**

### CODE REVIEW

File: src/code/audioMgr.c [hunk 1]
```diff
@@ -56,7 +56,7 @@ void AudioMgr_ThreadEntry(void* arg0) {
     osSyncPrintf("オーディオマネージャスレッド実行開始\n"); // "Start running audio manager thread"
     Audio_Init();
     AudioLoad_SetDmaHandler(DmaMgr_DmaHandler);
-    Audio_InitSound();
+    Audio_InitSfx();
```
Comment (Dragorn421) [inline src/code/audioMgr.c:59]: Is this really a "sfx" thing? It looks like it does more

File: include/variables.h [hunk 2]
```diff
@@ -200,8 +200,8 @@ extern volatile OSTime gRSPGFXTotalTime;
 extern volatile OSTime gRSPOtherTotalTime;
 extern volatile OSTime gRDPTotalTime;
 
-extern ActiveSound gActiveSounds[7][MAX_CHANNELS_PER_BANK]; // total size = 0xA8
-extern u8 gSoundBankMuted[];
+extern ActiveSfx gActiveSfxs[7][MAX_CHANNELS_PER_BANK]; // total size = 0xA8
```
[the remark this answers is not in the document]
Comment (fig02) [reply include/variables.h:203]: yea, would prefer ActiveSfx personally

File: src/code/audioMgr.c (hunk 1 above)
Comment (engineer124) [reply src/code/audioMgr.c:59]: It looks like 4 of the 6 are for sfxs, but yeah it has some general sequence stuff thrown in there. So I'll keep it as Sound

File: include/functions.h [hunk 3]
```diff
@@ -825,9 +825,9 @@ void Cutscene_HandleConditionalTriggers(PlayState* play);
 void Cutscene_SetSegment(PlayState* play, void* segment);
 void* MemCpy(void* dest, const void* src, s32 len);
 void GetItem_Draw(PlayState* play, s16 drawId);
-void SoundSource_InitAll(PlayState* play);
-void SoundSource_UpdateAll(PlayState* play);
-void SoundSource_PlaySfxAtFixedWorldPos(PlayState* play, Vec3f* worldPos, s32 duration, u16 sfxId);
+void SfxSource_InitAll(PlayState* play);
+void SfxSource_UpdateAll(PlayState* play);
+void SfxSource_PlaySfxAtFixedWorldPos(PlayState* play, Vec3f* worldPos, s32 duration, u16 sfxId);
```
[the remark this answers is not in the document]
Comment (engineer124) [reply include/functions.h:830]: I was a little hesitant to change this too tbh. `SoundSource` does roll off the tongue better, and the main request sfx function, `SfxSource_PlaySfxAtFixedWorldPos`, has a `sfx` in it without the prefix. But then this system as a whole is specifically about sfx's, so I'm not sure. Open to more opinions on it. I know @Roman971 was the one who named it in the first place and we had long discussions about this system before

File: src/overlays/actors/ovl_En_River_Sound/z_en_river_sound.c [hunk 4]
```diff
@@ -112,7 +112,7 @@ s32 EnRiverSound_FindClosestPointOnLineSegment(Vec3f* pointA, Vec3f* pointB, Vec
  * player.
  * Returns true if the distance between the `hearPos` and `soundPos` is less than 10000, false if not.
  */
-s32 EnRiverSound_GetSoundPos(Vec3s* points, s32 numPoints, Vec3f* hearPos, Vec3f* soundPos) {
+s32 EnRiverSound_GetSfxPos(Vec3s* points, s32 numPoints, Vec3f* hearPos, Vec3f* soundPos) {
```
Comment (Dragorn421) [inline src/overlays/actors/ovl_En_River_Sound/z_en_river_sound.c:115]:
```suggestion
s32 EnRiverSound_GetSfxPos(Vec3s* points, s32 numPoints, Vec3f* hearPos, Vec3f* sfxPos) {
```

**Review by Dragorn421 — APPROVED** [review]

File: include/functions.h (hunk 3 above)
[the remark this answers is not in the document]
Comment (Roman971) [reply include/functions.h:830]: I'm not a huge fan of SfxSource, but I guess it's probably okay

File: include/sfx.h [hunk 5]
```diff
@@ -9,7 +9,7 @@ typedef enum {
     /* 4 */ BANK_SYSTEM,
     /* 5 */ BANK_OCARINA,
     /* 6 */ BANK_VOICE
-} SoundBankType;
+}SfxBankType;
```
Comment (Dragorn421) [inline include/sfx.h:12]:
```suggestion
} SfxBankType;
```

**Review by fig02 — APPROVED** [review]
other than dragorns latest comment

**Review by Dragorn421 — APPROVED** [review]

**Review by Roman971 — APPROVED** [review]

**Comment by Dragorn421** [comment]
(conflicts sadge)

**Review by Dragorn421 — APPROVED** [review]

**Pull request merged**